.github/workflows/e2e_docker.yml

```diff
@@ -0,0 +1,65 @@
+name: E2E (docker)
```

Comment:
will there be other E2E tests? if not, "docker" here and the filename is noise

Reply:
i'll update the name